build.sbt

```
@@ -142,6 +147,14 @@ lazy val playJsonSettings = commonSettings ++ Seq(
   libraryDependencies += playJson
 )
 
+lazy val circeSettings = commonSettings ++ Seq(
+  libraryDependencies += circe,
+  libraryDependencies += circeAuto,
+  libraryDependencies += circeAutoExtras,
+  libraryDependencies += optionalEnumeratum,
+  libraryDependencies += circeParser
```

```suggestion
  libraryDependencies += circeParser % "test"
```
circe-parser module is used only in test, so we can add it as test only dependency